packages/jaeger-ui/src/actions/jaeger-api.test.js

```diff
@@ -49,11 +48,6 @@ describe('actions/jaeger-api', () => {
     expect(() => mock.verify()).not.toThrow();
   });
 
-  it('@JAEGER_API/FETCH_TRACE should return the promise', () => {
-    const { payload } = jaegerApiActions.fetchTrace(id);
-    expect(isPromise(payload)).toBeTruthy();
-  });
```

Comment:
Why are these tests removed? What is replacing them?

Reply:
> they are essentially type-checking, which should be TypeScript's job.

how is that type-checking achieved given that the source file in question is not in TS, but JS? 

Also

> Each of these fetchXYZ functions is essentially createAction wrapper 

That's an implementation detail of those functions, but the tests were enforcing the public API. Implementations may change.

Reply:
Ahh, my bad. I completely missed the `.js` extension. If we have to enforce the public API of these functions, then it would best to keep checking if we have a promise returned. I will revert the tests.